docs/index.md

```diff
@@ -0,0 +1,18 @@
+title: EthSigner
+description: EthSigner is an open-source Ethereum transaction signer developed under the Apache 2.0 license and written in Java.
+<!--- END of page meta data -->
+
+# EthSigner
+
+## What is EthSigner?
+
+//TODO
+
+## What can you do with EthSigner?
+
+EthSigner includes a [command line interface](Reference/EthSigner-CLI.md)
+//TODO
+
+## What does EthSigner support?
+
```

Comment:
Does this mean programming language? Java 8+ if so :P (Otherwise please ignore me!) 👍

Reply:
Same as previous comment, will be overridden by Madeline's work, so don't take this file into account.